linkerd/app/src/lib.rs

```diff
@@ -86,31 +86,32 @@ impl Config {
 
         let dns = dns.build();
 
-        let identity = info_span!("identity")
+        let identity = debug_span!("identity")
             .in_scope(|| identity.build(dns.resolver.clone(), metrics.control.clone()))?;
 
         let (drain_tx, drain_rx) = drain::channel();
 
-        let tap = info_span!("tap").in_scope(|| tap.build(identity.local(), drain_rx.clone()))?;
+        let tap = debug_span!("tap").in_scope(|| tap.build(identity.local(), drain_rx.clone()))?;
         let dst = {
             let metrics = metrics.control.clone();
             let dns = dns.resolver.clone();
-            info_span!("dst").in_scope(|| dst.build(dns, metrics, identity.local()))
+            debug_span!("dst").in_scope(|| dst.build(dns, metrics, identity.local()))
         }?;
 
         let oc_collector = {
             let identity = identity.local();
             let dns = dns.resolver.clone();
             let client_metrics = metrics.control;
             let metrics = metrics.opencensus;
-            info_span!("opencensus")
+            debug_span!("opencensus")
                 .in_scope(|| oc_collector.build(identity, dns, metrics, client_metrics))
         }?;
 
         let admin = {
             let identity = identity.local();
             let drain = drain_rx.clone();
-            info_span!("admin").in_scope(move || admin.build(identity, report, log_level, drain))?
+            debug_span!("admin")
```

Comment:
seems potentially worth keeping these top-level spans at INFO? i think it's mainly the ones in the actual data path where there's a noticeable latency impact?